routes/inventory.js

```diff
@@ -1,37 +1,81 @@
 const express = require("express");
 const router = express.Router();
 const fs = require("fs");
-const { v4: uuidv4 } = require('uuid');
+const { v4 } = require("uuid");
 
-require('dotenv').config();
-const {PORT, BACKEND_URL} = process.env;
+require("dotenv").config();
+const { PORT, BACKEND_URL } = process.env;
 
-
-// Read data file
-function readVideos() {
-    const videoData = fs.readFileSync("./data/warehouse.json");
-    const parsedData = JSON.parse(videoData);
-    return parsedData;
+// // Read data file
+function readInventory() {
+  const videoData = fs.readFileSync("./data/inventories.json");
```

Comment:
videoData should be replaced with a variable name like inventoryData

Reply:
Yeah Thanks! have updated and Merged.